content/blog/page-view-counter/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Using Azure Functions to create a page view counter
+date: "2019-12-26T22:12:03.284Z"
+author: nacho
+description:
+  "We can extend static sites by creating a simple page view counter that is processed serverless, and for free. Let's try to build it on Microsoft Azure services."
+tags:
+  - azure functions
+  - storage
+  - .NET
+  - cloud
+---
+
+## Introduction
+
+Since we are using [Gatsby](https://www.gatsbyjs.org/) to build this site as a set of static files, there are some common functionalities that you might find in dynamic sites which you will naturally be missing. However, this does not mean they cannot be implemented in some way.
+
+As a way of testing the development of a small, simple feature and some Azure offerings, we decided to try to build a page view counter for our blog posts.
```

Comment:
What is the exact concept that azure uses internally and its documentation and advertising to refer to a particular `service` such as `functions` and `whatever-the-name-of-the-service-that-provides-regular-vms` or `reddis`. If there is such a concept I would recommend we use that to improve SEO and also to improve the exact lang we use.

Bonus: just to make sure you know this if it makes any difference to your dev workflow, markdown paragraphs such as this can have newlines inside. The proper way of separating two paragraphs is by having _two_ newlines. 
Me personally I prefer a couple of lines for a single paragraph since it makes it easier to navigate the paragraph while editing and I thought it _might_ be useful to you (perdon por dar tantas vueltas para decir algo tan pelotudo)

Reply:
For the question related to Azure, the term is what I was using on the requirements section: "Compute" services. I did make sure to use documentation-aligned terms pretty much everywhere (it's also I'm something naturally used to).
For the markdown tip, good to know. I don't think it makes a difference to me while editing but it's still nice to know.

Reply:
👍